tangos/core/creator.py

```diff
@@ -10,11 +10,11 @@ class Creator(Base):
     __tablename__ = 'creators'
 
     id = Column(Integer, primary_key=True)
-    command_line = Column(String)
+    command_line = Column(String(128))
```

Comment:
I am not sure about this, in particular the `command_line` may be longer than 128 character. Should it be a `Text` instead?

Reply:
Yes I suppose it should be `Text`. In `sqlite` I think `TEXT` and `VARCHAR` are identical so this change should not cause any problems